tests/probes_test.go

```diff
@@ -25,6 +27,30 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 		tests.BeforeTestCleanup()
 	})
 
+	buildProbeBackendPodSpec := func(probe *v12.Probe) (*v1.Pod, func() error) {
```

Comment:
Why aren't you using `buildProbeBackendPodSpec` method introduced in the previous commit?

Reply:
I've deleted the method I've created previously, hope that makes it clearer (no point in having both).